internal/frr/frr.go

```diff
@@ -34,38 +33,24 @@ type Status struct {
 
 type FRR struct {
 	reloadConfig    chan reloadEvent
-	logLevel        string
```

Comment:
this was supposed to exist as a handle for changing teh configuration on the fly, but since we don't have such option today I guess it's better to remove it and reintroduce it when we have the option